docs/src/examples.md

```diff
@@ -2,543 +2,180 @@
+* `v`: _horizontal velocity (north-south component)_ measured in m/s.
+* `w`: _vertical velocity_ measured in m/s (set to zero in this example).
 
-#### Reconstructions
+#### Differentiation operators
 
-* ``I^c`` is the [face-to-center reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateF2C), called `first_order_If2c` in the example code.
-* ``I^f`` is the [center-to-face reconstruction operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.InterpolateC2F), called `first_order_Ic2f` in the example code.
-  - Currently this is just the arithmetic mean, but we will need to use a weighted version with stretched vertical grids.
-* ``U^f`` is the [center-to-face upwind product operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.Upwind3rdOrderBiasedProductC2F), called `third_order_upwind_c2f` in the example code
-  - This operator is of third-order of accuracy (when used with a constant vertical velocity and some reduced, but still high-order for non constant vertical velocity).
+Because this is a 1D vertical problem, we utilize the staggered vertical grid with:
 
+- `G_{c2f}` is the [gradient operator from cell centers to faces](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.GradientC2F), called `gradc2f` in the example code.
+- `D_{f2c}` is the [divergence operator from faces to centers](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.DivergenceF2C), called `divf2c` in the example code.
+- `A` is the [advection operator](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.AdvectionC2C), called `A` in the example code (though vertical advection is set to zero in this example).
 
-#### Differentiation operators
+#### Problem flow and set-up
 
-- ``D_h`` is the [discrete horizontal strong spectral divergence](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.Divergence), called `hdiv` in the example code.
-- ``wD_h`` is the [discrete horizontal weak spectral divergence](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.WeakDivergence), called `hwdiv` in the example code.
-- ``D^c_v`` is the [face-to-center vertical divergence](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.DivergenceF2C), called `vdivf2c` in the example code.
-  - This example uses advective fluxes equal to zero at the top and bottom boundaries.
-- ``G_h`` is the [discrete horizontal spectral gradient](https://clima.github.io/ClimaCore.jl/stable/operators/#ClimaCore.Operators.Gradient), called `hgrad` in the example code.
+This test case is set up in a vertical column domain `[0, L]` with height `L = 200m`. The Coriolis parameter is set to `f = 5e-5 s⁻¹`, the viscosity coefficient to `ν = 0.01 m²/s`, and the geostrophic wind to `(u_g, v_g) = (1.0, 0.0) m/s`.
```

Comment:
Hi @Snowdog85123 , you can actually use math mode here, i.e., `$` instead of code mode. Thank you!